README.md

```diff
@@ -0,0 +1,99 @@
+# LORIS MRI Downloader for the open PREVENT-AD dataset
+
+
+## 1. Description
+
+Due to the size (~200 GB) of the imaging dataset of the open PREVENT-AD dataset, 
+it is not possible to download the entire study’s imaging dataset via the browser. 
+However, various ways to download the imaging dataset are provided in this repository. 
+ 
+Users can choose one of the following tool to download the open PREVENT-AD images:
+  - a Python script called `LAMID_PREVENT-AD.py`
+  - Jupiter Notebook called `LAMID_PREVENT-AD.ipynb` 
+
+See section 2 below for the installation steps required for the tools available in 
+this repository.
+
+## 2. Installation and Requirements
+
+Requirements for the script:
+- Python 3 or higher
+- the following Python libraries: `getpass`, `json`, `requests`, `os`, `sys`, `getopt`
+and `errno`. 
+
+### Installation script (requires Ubuntu and sudo permission)
+
+An install script is provided in this repository to install Python and all its dependencies.
+
+Note that the script will only work on Ubuntu machines and requires sudo permission.
+
+To run the install script:
+```bash
+sudo bash install_LAMID_PREVENT-AD.sh
+```
+
+### Manual installation
+
+1. Install python 3 or higher with `pip`
```

Comment:
I would say ` and pip` instead of `with pip`. The former seems to imply that the installation. should be done using pip